index.bs

```diff
@@ -4558,6 +4558,23 @@ FIDO APIs use an alternative identifier for [=[RPS]=] called an |AppID|
 that identifier. Without this extension, they would need to be re-registered in
 order to be [=scoped=] to an [=RP ID=].
 
+In addition to setting the {{AuthenticationExtensionsClientInputs/appid}} extension input,
+using this extension requires some additional processing by the [=[RP]=]
+in order to [=authentication|authenticate=] U2F credentials:
```

Comment:
suggest:
in order to allow users to [=authentication|authenticate=] using their registered U2F credentials: